### ISSUE TITLE
deps: update winston deps, remove winston fixes

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2558]
The winston fixes were introduced to patch a memory leak in winston. PR merged into main repos, packages are now updated. patches no longer needed.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.32.5](https://github.com/open-wa/wa-automate-nodejs/compare/4.32.4...4.32.5) (2022-03-11)

- 🔥 cleanup: remove redundant winston patchfixes #2558 [`#2558`](https://github.com/open-wa/wa-automate-nodejs/issues/2558)
- Release 4.32.5 [`9eae61b`](https://github.com/open-wa/wa-automate-nodejs/commit/9eae61b03e5994d9ec0c7799dbed151a09d2aba0)
- 🔒️ fix critical npm warning [`b16ea47`](https://github.com/open-wa/wa-automate-nodejs/commit/b16ea4730dec0b997ffff056214a24a7bbaf824b)
- ⬆️ upgrade winston & winston-syslog deps #2558 [`0343f2c`](https://github.com/open-wa/wa-automate-nodejs/commit/0343f2c9ffbf043a5e91e3498233adb4783a75a9)
- updated types-only package [`f5b1aea`](https://github.com/open-wa/wa-automate-nodejs/commit/f5b1aeae5fe5ede3e8728d191b91cec0036c5f34)
